interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: `this` refers to the object in which something is being referenced
 
-  Researched answer:
+  Researched answer: `this` is an invoker in JavaScript, when an objects method or value is invoked `this` refers to the object and what it contains. 
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a library built for JavaScript to simplify the process in building web apps and user interfaces. 
 
-  Researched answer:
+  Researched answer: React.js is an open source JavaScript library used for creating user interfaces, single-page applications - specifically. 
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: this.state refers to the current step a component is at in its lifecycle
 
-  Researched answer:
+  Researched answer: Each component has its own lifecycle, it is the series of methods invoked during different stages of the components entire time invoked. 
```

Comment:
There is one particular lifecycle method that React class components require since React is always looking to _render_ or display something, the required method is the render method.